server/routerlicious/packages/services-core/src/mongoDatabaseManager.ts

```diff
@@ -43,7 +45,10 @@ export class MongoDatabaseManager implements IDatabaseManager {
     }
 
     private async getCollection<T>(name: string) {
-        const db = await this.mongoManager.getDatabase();
+        let db = await this.operationsDbMongoManager.getDatabase();
```

Comment:
If the condition below is met, then would we throw away the result of the operation awaited in this line? I think we should probably avoid this, since waiting on this kind of operation (especially if it involves I/O) could become expensive if executed often. Could we instead do something like:

```ts
 private async getCollection<T>(name: string) {
        const db = name === this.documentsCollectionName && this.globalDbEnabled ?
            await this.operationsDbMongoManager.getDatabase() :
            await this.operationsDbMongoManager.getDatabase();

        return db.collection<T>(name);
}        

```

Reply:
+1..small typo

`const db = name === this.documentsCollectionName && this.globalDbEnabled ? await this.globalDbMongoManager.getDatabase() : await this.operationsDbMongoManager.getDatabase();`

Reply:
Indeed, edited the comment and fixed the typo :)